### PR TITLE
JBPM-5424: Version selector does not work in the new form modeler.

### PR DESCRIPTION
**Pull request by pefernan**
Merge with: https://github.com/uberfire/uberfire/pull/607

### CODE REVIEW

**Comment by pefernan** [comment]
@wmedvede can you take a look please?

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/main/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorPresenter.java [hunk 1]
```diff
@@ -139,7 +139,9 @@ public void doLoadContent( FormModelerContent content ) {
         busyIndicatorView.hideBusyIndicator();
 
         // TODO: fix this, this return avoids to reload the layout editor again
```
Comment (wmedvede) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/main/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorPresenter.java]: is this comment still valid now that the "return" has gone?

**Comment by wmedvede** [comment]
just the silly comment above. 
Other than this, +1.

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/main/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorPresenter.java (hunk 1 above)
Comment (pefernan) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/main/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorPresenter.java]: @wmedvede you're right.

**Comment by pefernan** [comment]
@wmedvede fixed!

**Comment by pefernan** [comment]
@csadilek since https://github.com/uberfire/uberfire/pull/607 has been merged, can this PR be merged too?

**Pull request merged**